src/Data/Uniques/shield.lua

```diff
@@ -853,22 +800,20 @@ Variant: Pre 1.1.0
 Variant: Pre 2.0.0
 Variant: Pre 3.16.0
 Variant: Current
-Requires Level 68, 85 Str, 85 Int
 Implicits: 2
 {variant:1}+24% to all Elemental Resistances
 {variant:2,3,4}+12% to all Elemental Resistances
-{variant:1,2}+1 to Level of Aura Gems in this item
+{variant:1,2}+1 to Level of Socketed Aura Gems
 {variant:3,4}+2 to Level of Socketed Aura Gems
 Socketed Gems Cost and Reserve Life instead of Mana
-{variant:1,2,3}Socketed Gems have 50% increased Mana Reservation Efficiency
```

Comment:
This legacy mod still exists at 25%: https://www.pathofexile.com/trade/search/Standard/pWO8ojwS0

Reply:
The legacy modifier that shows 25% increased Reservation Efficiency is cosmetic only, it does not actually grant that value. I tested it myself with multiple legacy Prism Guardian shields, my Determination reserves 38.46% in the shield (50/130) instead of the expected 40% (50/125).